r/tests/testthat/test-compute.R

```diff
@@ -209,6 +213,7 @@ test_that("user-defined functions work during multi-threaded execution", {
   skip_if_not(CanRunWithCapturedR())
   skip_if_not_available("dataset")
   # Snappy has a UBSan issue: https://github.com/google/snappy/pull/148
+  # TODO(ARROW-17178): remove when user-defined function execution is stabilized
```

Comment:
Note that the skip is for snappy ubsan too, so both need to be resolved before we can unskip

Reply:
(See also ARROW-17283)